### PR TITLE
Docs for packaging as Docker image

### PR DESCRIPTION
**Pull request by alex-litvak**

### CODE REVIEW

**Comment by alex-litvak** [comment]
@rdallman @treeder Can you take a look please?

File: go/README.md [hunk 1]
```diff
@@ -4,39 +4,125 @@ This example will show you how to test and deploy Go (Golang) code to IronWorker
...
 
 ```sh
-docker run --rm -it -e "PAYLOAD_FILE=hello.payload.json" -v "$PWD":/app -w /app  iron/go ./hello
+iron docker login -e YOUR_DOCKER_HUB_EMAIL -u YOUR_DOCKER_HUB_USERNAME -p YOUR_DOCKER_HUB_PASSWORD
 ```
 
-And now it works, so let's package it up:
+Then just do everything the same as above.
+
+## If you don't want to package your code using Docker
```
Comment (rdallman) [inline go/README.md:99]: @treeder do we want to keep encouraging / documenting the old shove zip into docker image way ?

**Comment by rdallman** [comment]
awesome, thanks @alex-litvak it LGTM. see what @treeder thinks

**Comment by treeder** [comment]
Will review shortly.

File: go/README.md (hunk 1 above)
Comment (treeder) [inline go/README.md:99]: Not really, but maybe people already doing it this way... Maybe can move these into the wiki for this repo?

**Comment by treeder** [comment]
@alex-litvak looks great, but could you change the code examples for all of these to use the special env var that is passed in? Like the ruby one, it just prints it out. The examples should probably all use a custom env var and the payload in some way.

**Pull request merged**